lib/trocla/formats/cfssl.rb

```diff
@@ -0,0 +1,45 @@
+require 'json'
+require 'open3'
+class Trocla::Formats::Cfssl < Trocla::Formats::Base
+  def format(plain_password,options={})
+    #n o dig method on jruby 1.9 used by puppet ;/
```

Comment:
I don't understand this comment, should it read `no dig...`?

Reply:
Yes, I've used dig in previous iteration then discovered JRuby version used by puppet doesn't have it yet. Ill push a change with that fixed and requires moved inside the class. I let the comment there so nobody steps in same landmine as me